.github/workflows/sing-extension.yml

```diff
@@ -0,0 +1,73 @@
+# This workflow will perform signing of a plugin
+
+name: Sign Extension
+
+on:
+  workflow_run:
+    workflows: [ "CI Workflow" ]
+    types: [ completed ]
+    branches: [ "master" ]
+
+  workflow_dispatch:
+    inputs:
+      reason:
+        description: 'Reason for signing'
+        required: true
+
+jobs:
+  sign:
```

Comment:
we need then a job before this one that prepares node modules for archiver, should be simple and then exchange between jobs: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/storing-and-sharing-data-from-a-workflow#passing-data-between-jobs-in-a-workflow